js/theme.js

```diff
@@ -264,26 +264,29 @@ if (downloadNote.length >= 1) {
 
 //This code makes the Notes section of the Docs Left Nav collapsible
```

Comment:
Can you update this note to reflect your changes?

Reply:
> It looks like some of the link titles are being cut off:
> 
> <img alt="Screen Shot 2020-12-08 at 4 21 33 PM" width="364" src="https://user-images.githubusercontent.com/16585245/101543083-bffc4200-3971-11eb-8ced-4107b8d4a99b.png">
> 
Is this still causing an issue? I cleared my cache and it is showing that the link is separated on 2 lines